### DISCUSSION TITLE
Converting 2D lat/long to 1D lat/long for CF compliance

### DISCUSSION THREAD

**Discussion by rajkumar8581** [pyproj4/pyproj#1190] [General]
Hello all,
I am working on converting WRF output files to CF compliant netcdf files.

The lat long variables are supposed to be one dimensional arrays for CF compliance, as below:-

![image](https://user-images.githubusercontent.com/95094560/205108245-899972a6-515a-4c96-bcb4-c071050afc41.png)

The problem with wrfout files is the xlat/xlong are 2D arrays like below. Basically this hinders use of to_netcdf.

![image](https://user-images.githubusercontent.com/95094560/205108409-40143a9d-6aa3-4854-9294-93e4a5c42760.png)

**Comment by snowman2** [comment]
See:
- https://github.com/corteva/rioxarray/issues/209
- https://github.com/corteva/rioxarray/discussions/426
- https://github.com/xarray-contrib/xwrf
- https://github.com/NCAR/wrf-python
